Tasks/google-play-release/GooglePlay.js

```diff
@@ -308,12 +303,33 @@ function addAllChangelogs(languageCode, directory) {
             return pathIsFile;
         });
 
-        for (var i in changelogs) {
-            var fullChangelogPath = path.join(changelogDir, changelogs[i]);
+        var versionCodeFound = false;
+        changelogs.forEach(function(changelogFile) {
+            var changelogName = path.basename(changelogFile, path.extname(changelogFile));
+            var changelogVersion = parseInt(changelogName, 10);
+            if (apkVersionCodes.indexOf(changelogVersion) == -1) {
+                tl.debug(`File ${changelogFile} is not a valid version code`);
+                return;
+            }
+
+            versionCodeFound = true;
+            var fullChangelogPath = path.join(changelogDir, changelogFile);
             addAllChangelogsPromise = addAllChangelogsPromise.then(function (changelog) {
                 console.log(`Appending changelog ${changelog}`);
-                return addChangelog.bind(this, languageCode, changelog)();
+                return addChangelog.bind(this, languageCode, changelog, changelogVersion)();
             }.bind(this, fullChangelogPath));
+        });
+
+        // if there's a single change log file, and it doesn't match a version code, assume it's a global change log
+        if (!versionCodeFound && changelogs.length == 1) {
```

Comment:
I assume this is for your use case - using metadata with only one change log file instead of one for each versionCode? Is this a common practice?